src/hierarchy/Mammal.ts

```diff
@@ -1,14 +1,26 @@
 import { ICanJump } from "../interfaces/ICanJump";
 import Animal from "./Animal";
+import { AnimalCandidate } from "./Animal";
 
 export default abstract class Mammal extends Animal implements ICanJump {
     jump(): void {
         console.log(`${this.name} is jumping`);
     }
-    
-    
-    giveBirth(): void{
 
+
+    public giveBirth(): AnimalCandidate[] | void {
+        if (super.giveBirth()) {
+            this.timesGivenBirth++;
+            const candidatesforAnimals: AnimalCandidate[] = [];
+            const numberOfMammalBabies = 5;
```

Comment:
this constant can be at the top of mammal class and also upeprcase with nderscores.  NUMBER_OF_MAMMAL_BABIES